tips/tests/tests.md

```diff
@@ -0,0 +1,58 @@
+# Hoeveel tests moet ik maken.
+
+### ik wil voor elke rule een test maken. 
+##### Dat zijn 22 tests die gemaakt moeten worden.
+
+
+# Wat moet ik gaan testen.
+
+### Heeft geldige stadspas
+##### Als het product Minimafonds is en het besluit toekenning is, dan is het True.
+##### Als het product geen Minimafonds is en het besluit niet toekenning is, dan is het False.
```

Comment:
eigenlijk hoort hier ook nog een datum bij.

Reply:
oh op welke manier?

Reply:
Toekenning minder dan een jaar geleden. Een Stadspas is één jaar geldit. @drakonen klopt dat?

Reply:
Dat klopt. 1 jaar na toekenning.